app/composer.json

```diff
@@ -48,7 +48,15 @@
         "post-update-cmd": [
             "Illuminate\\Foundation\\ComposerScripts::postUpdate",
             "php artisan optimize"
-        ]
+        ],
+        "all": [
+            "@composer cs",
+            "@composer cs-tests"
+        ],
+        "cs": "./vendor/bin/phpcs --standard=phpcs.xml",
```

Comment:
Similar to how the phpunit test run is explained in app/readme.md, it would help to give a couple comments on how to run the phpcs commands to check coding style and fix coding style in app/readme.md.

If it is easy to do, it would be nice to add a PHPunit test that invokes phpcs so that when all the unit tests pass, it means both unit tests and coding style are all good.

Reply:
Sure. I will add it when we are using certain build system. TravisCI is the free service for the public project. I strongly recommend it to use it.